phpstan.neon

```diff
@@ -12,7 +12,7 @@ parameters:
     excludes_analyse:
         # Makes PHPStan crash
         - 'src/DependencyInjection/Configuration.php'
-        - 'src/Fixture/BrandFixture.php'
+        - 'src/Menu/BrandFormMenuBuilder.php'
```

Comment:
Maybe better to add something like:
```
- '/Method Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface\:\:dispatch\(\) invoked with 2 parameters, 1 required\./'
        - '/Parameter \#1 \$event of method Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface\:\:dispatch\(\) expects object, string given\./'
```
to `ignoreErrors` section instead so that other code is still analyzed by phpstan *and* we exclude similar errors when analyzing other (new) files